src/test/java/se/sundsvall/messaging/api/model/response/UserBatchesTest.java

```diff
@@ -0,0 +1,46 @@
+package se.sundsvall.messaging.api.model.response;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static se.sundsvall.messaging.TestDataFactory.createBatch;
+import static se.sundsvall.messaging.TestDataFactory.createPagingMetaData;
+
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import se.sundsvall.dept44.models.api.paging.PagingMetaData;
+import se.sundsvall.messaging.test.annotation.UnitTest;
+
+@UnitTest
```

Comment:
Why is this annotation necessary here?

Reply:
It is a heritage from the past. I'll add a task to remove all unit-test-annotations to clean things up.